core/gesture.js

```diff
@@ -743,8 +745,10 @@ class Gesture {
    */
   doBubbleClick_() {
     // TODO (#1673): Consistent handling of single clicks.
-    this.startBubble_.setFocus && this.startBubble_.setFocus();
-    this.startBubble_.select && this.startBubble_.select();
+    if (this.startBubble_ instanceof WorkspaceCommentSvg) {
```

Comment:
Why are workspace comments bubbles? That seems like not what they are.

Reply:
Notes for the curious:
I think it's just to make workspace comments similar to block comments. And block comments are icons/bubbles, just like mutators.
Bubbles sit on a different canvas than blocks, and always on top. They do that so that blocks on the workspace can never look like they're inside an open mutator.